kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -617,12 +617,13 @@
       </dependencies>
     </profile>    
     <profile>
+      <!-- temporary hack to let it test on jenkins id>wildfly14</id-->
       <id>wildfly11</id>
       <properties>        
         <kie.server.remoting.url>http-remoting://${container.hostname}:${container.port}</kie.server.remoting.url>
         <kie.server.context.factory>org.wildfly.naming.client.WildFlyInitialContextFactory</kie.server.context.factory>
         <org.kie.server.persistence.ds>java:/jdbc/jbpm</org.kie.server.persistence.ds>
-        <cargo.container.id>wildfly11x</cargo.container.id>
+        <cargo.container.id>wildfly14x</cargo.container.id>
```

Comment:
not sure there is cargo support for wildfly 14.. already but I guess @sutaakar would know best

Reply:
No, currently latest supported version is WildFly 13.